integration_tests/e2e/health.cy.ts

```diff
@@ -5,8 +5,8 @@ context('Healthcheck', () => {
       cy.task('stubAuthPing')
       cy.task('stubIncentivesApiPing')
       cy.task('stubPrisonApiPing')
-      cy.task('stubManageUser')
```

Comment:
dunno why i put my change to this on a different line 🙈 but it wasn't needed regardless :)